02-setup.md

```diff
@@ -32,7 +32,8 @@ $ git config --global color.ui "auto"
 
 (Please use your own name and email address instead of Dracula's. 
 This user name and email will be associated with your subsequent Git activity, 
-which means that any changes pushed to GitHub in a later lesson will include this information. 
+which means that any changes pushed to GitHub in a later lesson will include this information.
+We recommend setting up an account on [GitHub](https://github.com) if you haven't already and using this as your user.name so your computer works with this website in the future (including later in this lesson).
```

Comment:
I'm not sure I understand this. git's user.name is the "real name" of the person and not GitHub's login name. While it's possible to use GitHub's username as the name is your commits, in my experience this creates a lot of problems with attribution, and some people, including SWC, prefer having contributors for a real name in their commits